gallery/how_to/deploy_models/deploy_model_on_adreno.py

```diff
@@ -0,0 +1,298 @@
+
+name = "vgg16"
+model = gluon.model_zoo.vision.get_model(name, pretrained=True)
+input_name = "data"
+input_shape = (1, 3, 224, 224)
+shape_dict = {input_name: input_shape}
+mod, params = relay.frontend.from_mxnet(model, shape_dict)
+
+######################################################################
+# Precisions
+# ----------
+# Since TVM support Mixed Precision, we need to register mixed_precision_conversion:
+from  tvm.relay.op  import  register_mixed_precision_conversion
+
+conv2d_acc = "float32"
+
+@register_mixed_precision_conversion("nn.conv2d", level=11)
+def  conv2d_mixed_precision_rule(call_node: "relay.Call", mixed_precision_type: str):
+	global  conv2d_acc
+	return [
+		relay.transform.mixed_precision.MIXED_PRECISION_ALWAYS,
+		conv2d_acc,
+		mixed_precision_type,
+	] 
+
+@register_mixed_precision_conversion("nn.dense", level=11)
+def  conv2d_mixed_precision_rule(call_node: "relay.Call", mixed_precision_type: str):
+	global  conv2d_acc
+	return [
+		relay.transform.mixed_precision.MIXED_PRECISION_ALWAYS,
+		conv2d_acc,
+		mixed_precision_type,
+	]
+ 
+######################################################################
+# and also the conversion function itself
+def  convert_to_dtype(mod, dtype):
+	# downcast to float16
+	if  dtype == "float16"  or  dtype == "float16_acc32":
+		global  conv2d_acc
+		conv2d_acc = "float16"  if  dtype == "float16"  else  "float32"
+		from  tvm.ir  import  IRModule
+		mod = IRModule.from_expr(mod)
+		seq = tvm.transform.Sequential(
+			[
+				relay.transform.InferType(),
+				relay.transform.ToMixedPrecision()
+			]
+		)
+		with  tvm.transform.PassContext(opt_level=3):
+			mod = seq(mod)
+	return  mod
+
+######################################################################
+# Let's choose "float16_acc32" for example. 
+# You can also use "float16" or "float32" as other dtype options.
+dtype="float16_acc32"
+mod = convert_to_dtype(mod["main"], dtype)
+dtype = "float32"  if  dtype == "float32"  else  "float16"
+
+
+######################################################################
+# Load a test image
+# -----------------
+# As an example we would use classical cat image from ImageNet
+
+from  PIL  import  Image
+from  tvm.contrib.download  import  download_testdata
+
+img_url = "https://github.com/dmlc/mxnet.js/blob/main/data/cat.png?raw=true"
+img_path = download_testdata(img_url, "cat.png", module="data")
+img = Image.open(img_path)
+img = img.resize(shape_dict[input_name][2:])
+img = np.array(img) - np.array([123.0, 117.0, 104.0])
+img /= np.array([58.395, 57.12, 57.375])
+img = img.transpose((2, 0, 1))
+img = img[np.newaxis, :]
+
+######################################################################
+# Compile the model with relay
+# ----------------------------
+# Specify Adreno target before compiling to generate texture 
+# leveraging kernels and get all the benefits of textures 
+
+target="opencl -device=adreno"
+target_host="llvm -mtriple=arm64-linux-android"
```

Comment:
Should we add a description about `target_host`?

Reply:
BTW, this is depricated usage of the target/target host. Currently you have to create python object Target and initialize it by apprpriate target_host